app/javascript/components/Facets/Facets.tsx

```diff
@@ -1,9 +1,323 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { Accordion, DateRangePicker, Tag } from '@trussworks/react-uswds';
+
+import { StyleContext } from '../../contexts/StyleContext';
+import { FontsAndColors  } from '../SearchResultsLayout';
+
+import './Facets.css';
+
+const searchFilterSvgIcon = () => {
+  return (
+    <svg role="img" xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">
+      <title>Filter</title>
+      <path className="search-filer-icon" fill="#005EA2" d="M4.25 5.61C6.27 8.2 10 13 10 13v6c0 .55.45 1 1 1h2c.55 0 1-.45 1-1v-6s3.72-4.8 5.74-7.39A.998.998 0 0 0 18.95 4H5.04c-.83 0-1.3.95-.79 1.61z"/>
+    </svg>
+  );
+};
+
+const StyledWrapper = styled.div.attrs<{ styles: FontsAndColors; }>((props) => ({
+  styles: props.styles
+}))`
+
+  path.search-filer-icon {
+    fill: ${(props) => props.styles.buttonBackgroundColor};
+  }
+
+  .usa-accordion__button {
+    color: ${(props) => props.styles.searchTabNavigationLinkColor};
+  }
+
+  .serp-facets-wrapper .usa-tag{
+    color: ${(props) => props.styles.resultTitleColor};
+  }
+
+  .see-results-button {
+    background: ${(props) => props.styles.buttonBackgroundColor};
+  }
+
+  .clear-results-button{
+    color: ${(props) => props.styles.buttonBackgroundColor};
+  }
+`;
+
+type HeadingLevel = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6'; 
 
 export const Facets = () => {
+  const styles = useContext(StyleContext);
+
+  const audienceItems = [
+    {
+      title: 'Audience',
+      content: (
+        <fieldset className="usa-fieldset">
+          <div className="usa-checkbox">
+            <input
+              className="usa-checkbox__input"
+              type="checkbox"
+              name="audience"
+              value="small_business"
+              defaultChecked={true}
+            />
+            <label className="usa-checkbox__label">Small business <Tag>1024</Tag></label>
```

Comment:
Should we be using `i18n` for the label text?

Reply:
@ddzz currently locales are not setup for it. This will need to be handled separately.